foxpuppet/windows/browser/window.py

```diff
@@ -24,6 +25,19 @@ class BrowserWindow(BaseWindow):
     _tab_browser_locator = (By.ID, 'tabbrowser-tabs')
 
     @property
+    def tab_bar(self):
+        """Provide access to the Tab Bar.
+
+        Returns:
+            :py:class:`Tab_Bar`: FoxPuppet Tab Bar object.
+
+        """
+        window = BaseWindow(self.selenium, self.selenium.current_window_handle)
+        with self.selenium.context(self.selenium.CONTEXT_CHROME):
+            el = self.selenium.find_element(*self._tab_browser_locator)
```

Comment:
Is `_tab_browser_locator` used elsewhere? I'd suggest a rename to `_tab_bar_locator` for clarity.